packages/blitz/src/installer/utils/paths.ts

```diff
@@ -18,14 +18,28 @@ function getBlitzPath(type: string) {
   }
 }
 
+function getAppSourceDir() {
+  const srcPath = "src/pages"
+  const srcDir = fs.existsSync(path.resolve(srcPath))
+
+  if (srcDir) {
+    return "src"
+  } else {
+    return "app"
+  }
+}
+
 function findPageDir() {
   const srcPagePath = `src/pages`
-  const srcPage = fs.existsSync(path.resolve(srcPagePath))
+  const srcPage = getAppSourceDir()
 
-  if (srcPage) {
-    return srcPagePath
-  } else {
-    return `pages`
+  switch (srcPage) {
+    case "src": {
+      return srcPagePath
+    }
+    default: {
+      return `pages`
+    }
```

Comment:
TBH I am not sure if this change here helps the clarity. We cannot get rid of the const `srcPagePath`, one has to read another function to understand what is true/false; and the switch is harder to read for me than the if-else.

Reply:
I thought factoring out some logic would be helpful that's why I made a call to `getAppSourceDir` in `findPageDir` function.